pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -10,18 +11,31 @@
 
 
 class Requirement:
+    UNTIL_COMPARISON = re.compile(b'={2,3}|!=|~=|>=?|<=?')
+    UNTIL_SEP = re.compile(rb'\S[^;\s]+')
```

Comment:
is this different from `[^;\s]+`?

Reply:
Good call, I got confused parsing the `--hash` lines